### PR TITLE
Main page view

### PR DESCRIPTION
**Pull request by pabloderivia**
A 1st look of the Main Page (the green rectangle will be the ranking-board) and a simple footer component

### CODE REVIEW

File: npm-debug.log [hunk 1]
```diff
@@ -0,0 +1,45 @@
+0 info it worked if it ends with ok
```
[the remark this answers is not in the document]
Comment (pabloderivia) [reply npm-debug.log]: Honestly, I didn't even know this file was there

File: package.json [hunk 2]
```diff
@@ -1,26 +1,27 @@
...
-		"@vue/component-compiler-utils": "^3.0.0",
-		"vue-template-compiler": "^2.6.10"
-	}
+  "name": "myment-front",
+  "version": "1.0.0",
+  "description": "This repo contain the front API for the web application myment.",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "dev": "parcel src/index.html",
```
[the remark this answers is not in the document]
Comment (pabloderivia) [reply package.json]: It's a homeopathic test

File: src/components/LoggedHeader.vue [hunk 3]
```diff
@@ -0,0 +1,33 @@
+<template>
+    <div class="headerNav">
+        <img src="../assets/M.png" alt="our coolest logo"/>
```
[the remark this answers is not in the document]
Comment (pabloderivia) [reply src/components/LoggedHeader.vue]: Yep, I implemented the Image component after create this one. I will change it.

File: src/components/Main2.vue [hunk 4]
```diff
@@ -0,0 +1,59 @@
+<template>
+  <div>
+    <div class="welcome">
+      <span>Welcome to your plan!</span>
```
[the remark this answers is not in the document]
Comment (pabloderivia) [reply src/components/Main2.vue]: Yep, I didn't want just remove it, so I saved it like a simple template.

File: src/components/NewsBar.vue [hunk 5]
```diff
@@ -0,0 +1,46 @@
+<template> <!-- Just a cool way to show our news -->
+    <div class="containerBar marquee" >
+        <h6 >{{newNews}}</h6>
```
[the remark this answers is not in the document]
Comment (pabloderivia) [reply src/components/NewsBar.vue]: Mmm yep, you are right

File: src/VistaPruebaComponentes.vue [hunk 6]
```diff
@@ -0,0 +1,32 @@
+<template>
+    <div>
+        <html-clock theme="red"></html-clock>
```
[the remark this answers is not in the document]
Comment (pabloderivia) [reply src/VistaPruebaComponentes.vue]: Yep, a view for to test how  our components interact with other ones

File: design/.~lock.Mock-Up Project.pdf# [hunk 7]
```diff
@@ -0,0 +1 @@
+,pablo,pablo-PC,14.07.2019 20:43,file:///home/pablo/.config/libreoffice/4;
```
[the remark this answers is not in the document]
Comment (pabloderivia) [reply design/.~lock.Mock-Up Project.pdf#]: Yep, you are right. I will delete it

File: src/components/Button.vue [hunk 8]
```diff
@@ -0,0 +1,70 @@
...
+</template>
+
+<script>
+export default {
+
+  props: {
+    name: { //Generate, add or examine
+      type: String,
+      required: true,
+      default: 'generate'
```
[the remark this answers is not in the document]
Comment (pabloderivia) [reply src/components/Button.vue]: Ofc!

File: package.json (hunk 2 above)
[the remark this answers is not in the document]
Comment (EricDS-INFO) [reply package.json]: bro, use the webpack develop version

**Pull request merged**